scripts/prstate

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import os
+
+github = Github(os.environ['GITHUB_TOKEN'])
+
+for issue in github.search_issues(f"org:enarx state:open is:public is:pr"):
+    pr = issue.as_pull_request()
+
+    reviewers = {r.user.login for r in pr.get_reviews() if r.state == 'CHANGES_REQUESTED'}
```

Comment:
My concern with the `CHANGES_REQUESTED` query is that it will capture _every review that was ever submitted_ as `CHANGES_REQUESTED`. I don't know if this is a problem or not.